Public/Src/FrontEnd/Core/FrontEndHostController.cs

```diff
@@ -166,7 +166,8 @@ internal void SetState(FrontEndEngineAbstraction engine, IPipGraph pipGraph, ICo
             Engine = engine;
             FrontEndArtifactManager = CreateFrontEndArtifactManager();
             PipGraph = pipGraph;
-            
+            PipGraphFragmentManager = fragmentManager;

```

Comment:
>fragmentManager [](start = 38, length = 15)

Fragment manager can just be created here rather than passing it in right?

Reply:
I wasn't sure how to get the contexts needed to create it here.

---
In reply to: [288667119](https://github.com/ljcollins25/BuildXL/pull/1#discussion_r288667119) [](ancestors = 288667119)

Reply:
done

---
In reply to: [288692182](https://github.com/ljcollins25/BuildXL/pull/1#discussion_r288692182) [](ancestors = 288692182,288667119)